tests/core/fs/workspacefs/test_workspace_fs.py

```diff
@@ -438,3 +441,77 @@ async def mockup(*args):
     with running_backend.offline():
         with pytest.raises(FSBackendOfflineError):
             await alice_workspace.get_reencryption_need()
+
+
+@pytest.mark.trio
+async def test_upload_blocks(alice_user_fs, alice2_user_fs, running_backend, monkeypatch):
+    wid = await create_shared_workspace("w", alice_user_fs, alice2_user_fs)
+    alice_workspace = alice_user_fs.get_workspace(wid)
+
+    alice2_workspace = alice2_user_fs.get_workspace(wid)
+
+    fspath = "/taz"
+
+    await alice_workspace.touch(fspath)
+    await alice_workspace.sync()
+
+    await alice_workspace.write_bytes(fspath, b"aaaaaa" * DEFAULT_BLOCK_SIZE)
```

Comment:
`b"aaaaaa" * DEFAULT_BLOCK_SIZE` look strange: it is equivalent to `b"a" * 6 * DEFAULT_BLOCK_SIZE`, but `b"aaaaaa"` looks like it is made of an arbitrary number of characters.

What do you think of doing something like this instead ?
```python
# Fill the file with multiple blocks worth of data so that parallel upload will occurs
TAZ_V2_BLOCKS = 6
await alice_workspace.write_bytes(fspath, b"a" * DEFAULT_BLOCK_SIZE * TAZ_V2_BLOCKS)
```